src/main/java/io/jenkins/plugins/checks/github/status/GitHubSCMSourceStatusChecksTrait.java

```diff
@@ -26,6 +26,8 @@
     private String name = "Jenkins";
     private boolean suppressLogs = false;
     private boolean skipProgressUpdates = false;
+    private boolean publishConfluentIncPR = false;
+    private boolean PublishNonConfluentIncPR = false;
```

Comment:
nit: (but since you're matching the style here, more for your future knowledge) ... default value of a `boolean` field in Java is `false`, so it's generally bad style to specify it.